blackbox/docs/sql/reference/alter_table.txt

```diff
@@ -112,3 +121,56 @@ columns with a value each to identify the partition to alter.
 :value: The columns value.
 
 .. SEEALSO:: :ref:`Alter Partitioned Tables <partitioned_tables_alter>`
+
+
+Sub-commands
+============
+
+``REROUTE``
+-----------
+
+The ``REROUTE`` command provides different options to allocate a shard on a node.
+It explictly enforces and executes a reroute allocation on the cluster and makes
+it possible ot move a shard from one node to another or even allocate
+unnassigned shards to a node or cancel allocations.
+
+::
+
+    [ REROUTE reroute_option ]
```

Comment:
you need to provide all possibilities here, otherwise the shard_id and node_id descriptions below do not make sense